PendingReleaseNotes

```diff
@@ -253,6 +253,13 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   The 'cephfs-journal-tool --rank <fs_name>:<mds_rank> journal reset' and
   'cephfs-journal-tool --rank <fs_name>:<mds_rank> journal reset --force'
   commands require '--yes-i-really-really-mean-it'.
+* mgr/restful, mgr/zabbix: both modules, already deprecated since 2020, have been
```

Comment:
Should the release note be moved to `>=20.0.0` section (it's still in `>=19.0.0` section)?

Reply:
I'll leave up to @ktdreyer to decide if he's ok with v19 or v20?

Reply:
Well, v19 has shipped with these modules...  I'm not sure if dropping user-facing functionality like this in a stable release is acceptable.

Reply:
Anyone using Zabbix arguably has a very loose threshold of acceptability to begin with ;)